modules/javafx.controls/src/test/java/test/javafx/scene/control/SpinnerTest.java

```diff
@@ -390,6 +391,184 @@ public void expectExceptionWhenOneArgsDecrementCalled_noValueFactory() {
         assertEquals(1, (int) intValueFactory.getValue());
     }
 
+    // TODO : This should wrap around and select a value between min and max
+    @Test public void intSpinner_testWrapAround_increment_LargeStep() {
```

Comment:
I don't like the testing for known buggy behavior. If a partial fix is still planned, a better thing would be a test that verifies correct behavior that is `@Ignore`d pending the final fix.